src/components/Card.jsx

```diff
@@ -0,0 +1,9 @@
+import "./Card.css";
+
+export default function Card({ children, onClick }) {
+  return (
+    <>
+      <div className="card" onClick={onClick}>{children}</div>
+    </>
+  );
+}
```

Comment:
Ավելի կարճ, բայց ըստ նախընտրության՝ 

```suggestion
const Card = ({ children, onClick }) => (
   <div className="card" onClick={onClick}>
      {children}
   </div>
);

export default Card;
```
OR 
```suggestion
export const Card = ({ children, onClick }) => (
   <div className="card" onClick={onClick}>
      {children}
   </div>
);
```